docs/vcpkg_registry_release_process.md

```diff
@@ -0,0 +1,14 @@
+# vcpkg_registry_release_process
+
+This document describes the acceptance criteria / process we use when doing a vcpkg registry release.
+
+1. Run vcpkg z-changelog `<SHA OF LAST RELEASE>` > `path/to/results.md`
+1. Create a new GitHub release in the registry repo.
+1. Run 'auto generate release notes'
+1. Change `## New Contributors` to `#### New Contributors`
+1. Copy the contents to the end of `path/to/results.md` (the `#### New Contributors` part should line up)
+1. Change the link to the most recent CI build, and copy number of successful ports into the table at the top.

```

Comment:
This seems likely to cause a mistake (forget to fill in the correct final table); I'd drop it.

Reply:
Are you saying to drop the table entirely? I think @AugP said he liked the table...